iac/terraform/aws/acm.tf

```diff
@@ -0,0 +1,11 @@
+data "aws_acm_certificate" "certificate" {
+  domain      = var.bloodconnect_domain
+  statuses    = ["ISSUED"]
+  most_recent = true
+  provider = aws.virginia
```

Comment:
format terraform files, check documentation on how we are formatting terraform

Reply:
Is it possible to create a way so that every time we commit it executes the formatting command?

Reply:
https://medium.com/nerd-for-tech/how-to-auto-format-hcl-terraform-code-in-visual-studio-code-6fa0e7afbb5e